.npmignore

```diff
@@ -0,0 +1,3 @@
+src
+node_modules
+dist/tsbuildinfo
```

Comment:
npm will already use the `.gitignore` for ignoring files to publish, why is this here?